api/rest/v2/observation.md

```diff
@@ -10,95 +10,192 @@ permalink: /api/rest/v2/observation
 
 # /v2/observation
 
-This API fetches statistical observations. An observation is associated with an
-entity and variable at a particular date. For example, “population of USA in
-2020”, “GDP of California in 2010”, “predicted temperature of New York in 2050”,
-and so on.
+The Observation API fetches statistical observations. An observation is associated with an
+entity and variable at a particular date: for example, "population of USA in
+2020", "GDP of California in 2010", and so on.
 
-When querying observations, you need to provide variables, entities, and date.
-Variables are specified as a list in the form of
+## Request
 
-```json
+<div class="api-tab">
+  <button id="get-button" class="api-tablink" onclick="openTab(event, 'GET-request')">
+    GET request
+  </button>
+  <button id="post-button" class="api-tablink" onclick="openTab(event, 'POST-request')">
+    POST request
+  </button>
+</div>
+
+<div id="GET-request" class="api-tabcontent api-signature">
+https://api.datacommons.org/v2/observation?key=AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI&date=<var>DATE_EXPRESSION</var>&variable.dcids=<var>DCID_LIST</var>&entity.dcids|expression=<var>DCID_LIST_OR_RELATION_EXPRESSION</var>&select=variable&select=entity[&select=value][&select=date]
+</div>
+
+<div id="POST-request" class="api-tabcontent api-signature">
+URL:
+https://api.datacommons.org/v2/observation
+
+Header:
+X-API-Key: AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI
+
+JSON data:
 {
-  "dcids": ["<variable_dcid_1>", "<variable_dcid_2>"]
+  "date": "<var>DATE_EXPRESSION</var>",
+  "variable.dcids": [
+      "<var>VARIABLE_DCID_1</var>",
+      "<var>VARIABLE_DCID_2</var>",
+      ...
+    ],
+  "entity.dcids": [
+      "<var>ENTITY_DCID_1</var>",
+      "<var>ENTITY_DCID_2</var>",
+      ...
+    ],
+  "entity.expression": <var>ENTITY_EXPRESSION</var>,
+  "select": "variable",
+  "select": "entity",
+  ...
 }
-```
+</div>
+
+<script src="/assets/js/syntax_highlighting.js"></script>
+<script src="/assets/js/api-doc-tabs.js"></script>
+
+### Query parameters
```

Comment:
wonder if we should list query parameters either: 
a. alphabetically
b. required -> optional

Reply:
It's an interesting idea; but I prefer doing it in the order that kind of makes sense logically (which is what I've tried to do here).